executor/common_linux.h

```diff
@@ -725,6 +757,30 @@ static void initialize_devlink_pci(void)
 #endif
 #endif
 
+#if __NR_syz_80211_join_ibss || __NR_syz_80211_inject_frame || SYZ_EXECUTOR || SYZ_NET_DEVICES
```

Comment:
Done. 

Unfortunately, such replacement was only possible in this specific line. I haven't found a way to tell syzkaller never to generate a C program with `__NR_syz_80211_join_ibss` or `__NR_syz_80211_inject_frame`, but without `SYZ_WIFI`. There is [one mechanism](https://github.com/google/syzkaller/blob/master/pkg/csource/csource.go#L210), but it doesn't seem to prevent `__NR*` defines, and this leads to warnings about unused functions.